package.json

```diff
@@ -48,5 +48,8 @@
     "react-hot-loader": "^3.0.0-beta.6",
     "webpack": "^1.12.14",
     "webpack-dev-server": "^1.16.2"
+  },
+  "dependencies": {
+    "prop-types": "^15.5.8"
```

Comment:
This should actually be a peer dependency.  Sorry I didn't catch this before.

Someone that's using this project already has a React project set up and should already have `prop-types` installed, so they don't want it bundled into this module.  See [this line in webpack.config.js](https://github.com/justinmc/react-audio-player/blob/master/webpack.config.js#L11), where I'm excluding all modules from the build.